### PR TITLE
Add more complete IV examples. Edit pvp rank example text.

### PR DESCRIPTION
**Pull request by versx**
IV/Level:
image

PVP:
image

### CODE REVIEW

File: static/js/index.js [hunk 1]
```diff
@@ -5159,7 +5159,7 @@ function getTimeSince (date) {
     return str;
 }
 
-const ivFilterPrompt = 'Please enter an IV Filter. Example: (S0-1 & A15 & D15 & (CP1400-1500 | CP2400-2500)) | L35 | 90-100';
+const ivFilterPrompt = 'Please enter an IV and/or Level Filter. Examples:\n(A0-1 & D15 & S15 & (CP1400-1500 | CP2400-2500)) | L34-35 | 90-100\n!0-100\nA0 & D0 & S0 | 100';
```
Comment (sketchysocks) [inline static/js/index.js:5162]:
```suggestion
const ivFilterPrompt = 'Please enter an IV and/or Level Filter. Examples:\n(A0-1 & D15 & S15 & (CP1400-1500 | CP2400-2500)) | L34-35 | 90-100\nShows all Pokemon that are between CP1400-1500 & CP2400-2500, level 34 or 35, have 0 or 1 Attack, 15 Defense/Stamina, and total IVs are between 90 and 100%. \n!0-100\nDisplays Pokemon that do not have their IVs checked\nA0 & D0 & S0 | 100\nResults in all 0% or 100% Pokemon';
```

File: static/js/index.js [hunk 2]
```diff
@@ -5227,7 +5227,7 @@ function manageGlobalIVPopup (id, filter) {
 }
 
 function manageGlobalPVPPopup (id, filter) {
-    const result = prompt('Please enter a PVP Filter. Example: 1-5', filter['pvp_' + id].filter);
+    const result = prompt('Please enter a PVP rank Filter. Example: 1-5', filter['pvp_' + id].filter);
```
Comment (sketchysocks) [inline static/js/index.js:5230]:
```suggestion
    const result = prompt('Please enter a PVP rank Filter. Example: 1-5\nWill display all Pokemon that are rank 1-5 for Great and Ultra League.', filter['pvp_' + id].filter);
```

File: static/js/index.js (hunk 1 above)
Comment (versx) [reply static/js/index.js:5162]: The issue with this is that it adds more lines to the alert box, which might not be a problem, but smaller devices _might_ have an issue. I'll test the changes right now.

Comment (sketchysocks) [reply static/js/index.js:5162]: Let me know. I think showing what all of that actually results in can be helpful for the users to process it, but fine to save it for a ui redo at some point if it's messy.

Comment (versx) [reply static/js/index.js:5162]: Just a bit overwhelming lol
image

Comment (clburlison) [reply static/js/index.js:5162]: this recommendation is too much for this UI element. 

💡 how about a "Help" button next to configure that links to [https://wwm.readthedocs.io/projects/mapjs/en/latest/filtering](). From there we can include 3-4 examples + english text on what it is actually doing?

**Review by clburlison — APPROVED** [review]
It's a bunch of text for this dialog box but an improvement overall

**Comment by sketchysocks** [comment]
Now that we have #217, maybe we should use these spots to explain the difference between IV-OR and IV-AND?

**Pull request closed without merging**